lib/internal/Magento/Framework/HTTP/PhpEnvironment/Request.php

```diff
@@ -696,7 +696,7 @@ public function getHeader($name, $default = false)
         if ($header instanceof HeaderInterface) {
             return $header->getFieldValue();
         }
-        return false;
+        return  $header;
```

Comment:
Pls fix two spaces

Reply:
@ihor-sviziev Thanks for check. Updated PR